scripts/setup_test.py

```diff
@@ -35,6 +35,7 @@
 
 RELEASE_TEST_DIR = os.path.join('core', 'tests', 'release_sources', '')
 MOCK_TMP_UNTAR_PATH = os.path.join(RELEASE_TEST_DIR, 'tmp_unzip.tar.gz')
+MOCK_YARN_PATH = os.path.join(RELEASE_TEST_DIR, 'yarn-' + common.YARN_VERSION)
```

Comment:
Not sure if the mock directory should go to `release_sources`? Maybe change to `core/tests/data`?

Reply:
Done.